name-segments/nouns/urban.txt

```diff
@@ -0,0 +1,17 @@
+metro
+metropolitan
+urbanite
+city
+urbano
+citylights
+streetsahead
+road
+skyhigh
```

Comment:
This word, and some others in the file aren't Nouns. Just "sky" is though.

Here's the words in an example name:
Eve Black**skyhigh**
Eve Black**sky**

Please look over the words you've submitted and try to fix please